usaon_vta_survey/constants/sba.py

```diff
@@ -26,18 +26,365 @@
+        ],
+        'Future Projections of Community Changes': [
+            'Characterize the magnitudes and rates of Arctic system'
+            ' changes and their impacts on communities',
+            'Improve the projections of impacts from Arctic system'
+            ' changes on communities',
+        ],
+    },
+    'Sociocultural Services': {
+        'Cultural and Spiritual Experiences': [
+            'Ensure continued access to opportunities for recreation'
+            ' and human connection with nature',
+            'Maintain areas of cultural significance in the Arctic',
+            'Maintain the vitality of Arctic languages, cultures,'
+            ' and communities to preserve knowledge sources',
+        ],
+        'Knowledge Development and Integration': [
+            'Ensure integration of indigenous languages, cultures, and'
+            ' communities for knowledge co-production',
+            'Improve understafing of Arctic processes and cultures to'
+            ' improve and enhance knowledge co-production',
+            'Support development, co-production, and dissemination of'
+            ' scientific knowledge across Arctic cultures',
+        ],
+        'Socioeconomics': [
+            'Improve understanding of formal and informal exchange networks'
+            ' for Arctic resources',
+            'Improve understanding of socioeconomic systems that impact the Arctic',
+            'Improve understanding of the cryospheric and environmental'
+            ' processess in the Arctic on socioeconomic systems',
+        ],
+    },
+    'Terrestrial and Freshwater Ecosystems and Processes': {
+        'Terrestrial and Freshwater Ecosystem Biodiversity': [
+            'Identify and preserve culturally important terrestrial'
+            ' and freshwater areas',
+            'Identify and preserve ecologically sensitive terrestrial'
+            ' and freshwater areas for biodiversity',
+            'Identify and understand the diversity of biota throughout their ranges',
+            'Manage and preserve biota throughout their ranges',
+            'Promote resilience and sustainability of biodiversity in'
+            ' terrestrial and freshwater ecosystems',
+        ],
+        'Terrestrial and Freshwater Ecosystem Responses to Arctic Changes': [
+            'Improve understanding of changing environmental impacts'
+            ' on terrestrial and freshwater ecosystems',
+            'Improve understanding of ecological and evolutionary responses'
+            ' of terrestrial and freshwater organisms to changes in the Arctic',
+            # 'Inform human adaptation to ecosystem changes',
+            'Manage disturbances to terrestrial and freshwater ecosystems',
+        ],
+        'Terrestrial and Freshwater Living Resources': [
+            'Assess and manage land cover and land use in a sustainable manner',
+            # 'Characterize and assess the status and trends of Arctic and'
```

Comment:
```sqlalchemy.exc.IntegrityError: (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "pk_societal_benefit_key_objective"

DETAIL:  Key (id)=(Manage Arctic and migratory living resources in a sustainable manner) already exists```

These gave the error pasted above because the exact same value exists under another sub-area. Not sure what the best method is to proceed here. - maybe asking Hazel?